src/components/addTranslation/AddTranslation.tsx

```diff
@@ -21,11 +21,11 @@ const mapState = (state: IState) => {
 };
 
 const validationSchema = Yup.object().shape({
-  name: Yup.string()
+  title: Yup.string()
     .required("Title is required")
     .min(3),
   isEmail: Yup.boolean(),
-  author: Yup.string(),
+  author: Yup.string().required("Title is required"),
```

Comment:
You sure this required is in the proper place? :P

Reply:
:P